<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>مدیریت کارها - Qt UI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="inputLayout">
     <item>
      <widget class="QLineEdit" name="taskInput">
       <property name="placeholderText">
        <string>عنوان کار...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="priorityBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="categoryBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>افزودن</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLineEdit" name="searchInput">
       <property name="placeholderText">
        <string>جستجو...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterPriority"/>
     </item>
     <item>
      <widget class="QComboBox" name="filterCategory"/>
     </item>
     <item>
      <widget class="QPushButton" name="themeToggleButton">
       <property name="text">
        <string>🌙 تغییر تم</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="taskList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
